src/median.js

```diff
@@ -22,9 +21,7 @@ var medianSorted = require('./median_sorted'),
  * median(incomes); //= 3.5
  */
 function median(x /*: Array<number> */)/*:number*/ {
-    // Sorting the array makes it easy to find the center, but
-    // use `.slice()` to ensure the original array `x` is not modified
-    return medianSorted(numericSort(x));
+    return +quantile(x, 0.5);
```

Comment:
Yes, because `quantile` returns either a number or an array, but we're only interested in the `number` case here.